src/components/molecules/inputAndLabel/InputAndLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import InputTextArea from '../../atoms/inputTextArea/InputTextArea'
+import DatePicker from '../../atoms/datePicker/DatePicker'
+import Label from '../../atoms/label/Label'
+import InputText from '../../atoms/inputText/InputText'
+
+function InputAndLabel ({ inputType = 'text', label = '', id,  placeholder = '', className, fullWidth }) {
```

Comment:
```suggestion
function InputAndLabel ({ 
  inputType = 'text',
  label = '', id, 
  placeholder = '',
  className,
  fullWidth 
}) {
```
peut etre indenté comme ca quand il y a beaucoup de props